microprofile/WFLY-12770_Create_quickstart_for_MicroProfile_Fault_Tolerance.adoc

```diff
@@ -0,0 +1,78 @@
+= WFLY-12770 Create quickstart for MicroProfile Fault Tolerance
+:author:            Radoslav Husar
+:email:             rhusar@redhat.com
+:toc:               left
+:icons:             font
+:keywords:          quickstarts,microprofile,fault-tolerance
+:idprefix:
+:idseparator:       -
+
+== Overview
+
+The goal is to add a new Quickstart demonstrating how to make use of https://microprofile.io/project/eclipse/microprofile-fault-tolerance[Eclipse MicroProfile Fault Tolerance]
+within the WildFly application server.
+There is no container-specific integration in place, thus this quickstart will be kept minimal.
+
+== Issue Metadata
+
+=== Issue
+
+* https://issues.redhat.com/browse/WFLY-12770[WFLY-12770 Create quickstart for MicroProfile Fault Tolerance 2.1]
+* https://issues.redhat.com/browse/EAP7-1370[EAP7-1370 Quickstart for MP Fault Tolerance 2.1]
+
+=== Related Issues
+
+* https://issues.redhat.com/browse/WFLY-13024[WFLY-13024 Integrate MP Fault Tolerance 2.1]
+
+=== Dev Contacts
+
+* mailto:rhusar@redhat.com[Radoslav Husar]
+
+=== QE Contacts
+
+* mailto:tterem@redhat.com[Tomas Terem]
+
+=== Testing By
+
+[ ] Engineering
```

Comment:
This RFE should be tested by Engineering.

Reply:
Done.